app/test_main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import pytest
+from unittest import mock
+
+from app.main import can_access_google_page
```

Comment:
Imports are ordered incorrectly, follow PEP-8 import rules
```suggestion
from unittest import mock

import pytest

from app.main import can_access_google_page
```